Examples/Tests/reduced_diags/inputs

```diff
@@ -79,6 +79,19 @@ MR.type = RhoMaximum
 MR.intervals = 200
 NP.type = ParticleNumber
 NP.intervals = 200
+FR_Max.type = FieldReduction
+FR_Max.intervals = 200
+FR_Max.reduced_function(x,y,z,Ex,Ey,Ez,Bx,By,Bz) = sqrt(Bx**2 + By**2 + Bz**2)
+FR_Max.reduction_type = Maximum
+FR_Min.type = FieldReduction
+FR_Min.intervals = 200
+FR_Min.reduced_function(x,y,z,Ex,Ey,Ez,Bx,By,Bz) = x*Ey*Bz
+FR_Min.reduction_type = Minimum
+FR_Integral.type = FieldReduction
+FR_Integral.intervals = 200
+FR_Integral.reduced_function(x,y,z,Ex,Ey,Ez,Bx,By,Bz) = "((y > 0)*(z < 0)) *
+                              (0.5*((Ex**2 + Ey**2 + Ez**2)*epsilon0+(Bx**2 + By**2 + Bz**2)/mu0))"
```

Comment:
Shall we try to use the new `if` feature available with the parser (see #1976)? Would that work here?
Just to use it in as many input files as possible, so that developers and users start getting used to it.
Could be done also in a separate PR, of course, but maybe it is a quick change.

Reply:
Would look like this:
```suggestion
FR_Integral.reduced_function(x,y,z,Ex,Ey,Ez,Bx,By,Bz) = "if(y > 0 and z < 0,
                              0.5*((Ex**2 + Ey**2 + Ez**2)*epsilon0+(Bx**2 + By**2 + Bz**2)/mu0), 0)"
```